prac_08/box_layout_demo.py

```diff
@@ -0,0 +1,21 @@
+from kivy.app import App
+from kivy.lang import Builder
+
+
+class BoxLayoutDemo(App):
+    def build(self):
+        self.title = "Box Layout Demo"
+        self.root = Builder.load_file('box_layout.kv')
+        return self.root
+
+    def handle_greet(self):
+        """On press greet using inputted name"""
+        self.root.ids.output_label.text = "Hello " + self.root.ids.input_name.text
+
+    def handle_clear(self):
+        """On press set text fild and label to blank"""
+        self.root.ids.output_label.text = ""
```

Comment:
Could change output to "Enter Name" so the window is set back to how it was initially